src/createPurgeCssPlugin.js

```diff
@@ -8,11 +8,15 @@ const extractAllSelectorLikeStrings = class {
     }
 };
 
-module.exports = options =>
-    new PurgecssPlugin(
+module.exports = options => {
+    const globs = glob
+        .sync(options.globs, { mark: true })
+        .filter(function(f) { return !/\/$/.test(f); });
```

Comment:
Could you use an arrow function here?

Reply:
@sebastiandedeyne Of course 👍 Changed it in the last commit